cli_tools/gce_windows_upgrade/upgrade_script_2008r2_to_2012r2.ps1

```diff
@@ -43,12 +43,19 @@ online disk noerr
   }
 
   # Find the drive which contains install media.
-  $Drives = Get-WmiObject Win32_LogicalDisk
-  ForEach ($Drive in $Drives) {
-    if (Test-Path "$($Drive.DeviceID)\Windows_Svr_Std_and_DataCtr_2012_R2_64Bit_English") {
-      $script:install_media_drive = "$($Drive.DeviceID)"
+  For ($i=0; $i -lt 5; $i++) {
+    $Drives = Get-WmiObject Win32_LogicalDisk
+    ForEach ($Drive in $Drives) {
+      if (Test-Path "$($Drive.DeviceID)\Windows_Svr_Std_and_DataCtr_2012_R2_64Bit_English") {
+        $script:install_media_drive = "$($Drive.DeviceID)"
+      }
     }
+    if ($script:install_media_drive){break}
```

Comment:
For consistency and readability it's better for all condition statement to follow the same formatting where the "then" is on the next line.

```suggestion
    if ($script:install_media_drive) {
      break
    }
```